src/library/io/mm-reader.cpp

```diff
@@ -569,7 +569,7 @@ clsparseSCsrMatrixfromFile(clsparseCsrMatrix* csrMatx, const char* filePath, cls
     MatrixMarketReader< cl_float > mm_reader;
     if( mm_reader.MMReadFormat( filePath ) )
         return clsparseInvalidFile;
-
+#if 0
```

Comment:
This code was likely disabled to bypass the bug raised in #144. The change should be reverted, since PR #154 solved the underlying problem.